ci/gcov/coverage-generator.sh

```diff
@@ -49,6 +49,6 @@ mv -f usr/share/nnstreamer/unittest/result/* ../gcov_html
 popd
 
 # Remove unused folders
-rm -rf $dirpath/local
-rm -rf $dirpath/usr
+sudo rm -rf $dirpath/local
```

Comment:
I am not sure that we have to use "sudo" command to remove the existing folders.

Reply:
Please append annotation as follows.
```
#  Remove ununsed folder. Note that we have to use 'sudo' command  to remove these files because 'gbs' command creates some files with "root' ID via "sudo chroot" operation. 
```

Reply:
fixed